.github/workflows/post-merge-deploy.yml

```
@@ -0,0 +1,48 @@
+name: Package for Deployment
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    name: Package for build
```

```suggestion
    name: Publish
```
This now publishes to both dev and build